dest/independentTests/projects.spec.ts

```diff
@@ -0,0 +1,61 @@
+import test, { expect, Page } from "@playwright/test";
+import { projectPageData } from "../../Data/projectsPage.data";
+import { factory } from "../../factory";
+import { v4 as uuid } from 'uuid';
+import homePage from "../../pages/home.page";
+import { profilePage } from "../../pages/profile.page";
+import { projectPage } from "../../pages/project.page";
+import { utils } from "../../Utils/Utils";
+
+test.describe('projects tests', () => {
+    let page: Page;
+    let util: utils;
+    let homePage: homePage;
+    let profilePage: profilePage
+    let projectPage: projectPage;
+
+    let previousNumberOfProject: number;
+
+    test.beforeAll(async ({ browser }) => {
+        var context = await browser.newContext();
+        page = await context.newPage();
+
+        util = factory.initUtils(page);
+        homePage = factory.initHomePage(page);
+        profilePage = factory.initProfilePage(page);
+        projectPage = factory.initProjectsPage(page);
+
+        await homePage.loadHomePage();
+    });
+
+    test.beforeEach(async () => {
+        await profilePage.clickProfilePage();
+        await profilePage.clickProjectsSectionLink();
+        previousNumberOfProject = await profilePage.getNumberOfProjects();
+        await profilePage.clickCreateProject();
```

Comment:
why is this in beforeEach?
the create project is called based on need for the tests, in case of it being a prerequisite for a test, it should be called there, not in before each test